tests/integration/components/product-destroy/component-test.js

```diff
@@ -0,0 +1,42 @@
+import { expect } from 'chai';
+import { describe, it, beforeEach } from 'mocha';
+import { setupComponentTest } from 'ember-mocha';
+import hbs from 'htmlbars-inline-precompile';
+import { find, click } from 'ember-native-dom-helpers'; // different find
+import sinon from 'sinon';
+
+describe('Integration | Component | product destroy', function() {
```

Comment:
this should be part of `confirm-action` component (we do not test here if the something has been destroy

(still product destroy component also deserve at least one test :-)